{"uid":"8b0b505c94856499","name":"Unsuccessful Login","fullName":"Login functionality for GuviZenClass portal: Unsuccessful Login","historyId":"07c7955c108e91d7b183d93221352f2c","time":{"start":1753527199725,"stop":1753527205569,"duration":5844},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=138.0.7204.169)\nStacktrace:\n\tGetHandleVerifier [0x0x7ff66a13e935+77845]\n\tGetHandleVerifier [0x0x7ff66a13e990+77936]\n\t(No symbol) [0x0x7ff669ef9cda]\n\t(No symbol) [0x0x7ff669ed21a1]\n\t(No symbol) [0x0x7ff669f7fc6e]\n\t(No symbol) [0x0x7ff669fa0432]\n\t(No symbol) [0x0x7ff669f786a3]\n\t(No symbol) [0x0x7ff669f41791]\n\t(No symbol) [0x0x7ff669f42523]\n\tGetHandleVerifier [0x0x7ff66a41684d+3059501]\n\tGetHandleVerifier [0x0x7ff66a410c0d+3035885]\n\tGetHandleVerifier [0x0x7ff66a430400+3164896]\n\tGetHandleVerifier [0x0x7ff66a158c3e+185118]\n\tGetHandleVerifier [0x0x7ff66a16054f+216111]\n\tGetHandleVerifier [0x0x7ff66a1472e4+113092]\n\tGetHandleVerifier [0x0x7ff66a147499+113529]\n\tGetHandleVerifier [0x0x7ff66a12e298+10616]\n\tBaseThreadInitThunk [0x0x7ff976d1e8d7+23]\n\tRtlUserThreadStart [0x0x7ff977f3c34c+44]\n\n","statusTrace":"  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features\\steps\\login_steps.py\", line 47, in step_invalid_login\n    context.login_page.enter_username(\"invalid@example.com\")\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\OneDrive\\Desktop\\task18\\features\\pages\\login_page.py\", line 25, in enter_username\n    self.enter_text(self.USERNAME_INPUT, username)\n    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\OneDrive\\Desktop\\task18\\features\\pages\\base_page.py\", line 42, in enter_text\n    element = self.wait_for_element(locator)\n  File \"C:\\Users\\surek\\OneDrive\\Desktop\\task18\\features\\pages\\base_page.py\", line 22, in wait_for_element\n    return self.wait.until(EC.presence_of_element_located(locator))\n           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 137, in until\n    value = method(self._driver)\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\expected_conditions.py\", line 110, in _predicate\n    return driver.find_element(*locator)\n           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 898, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=138.0.7204.169)\nStacktrace:\n\tGetHandleVerifier [0x0x7ff66a13e935+77845]\n\tGetHandleVerifier [0x0x7ff66a13e990+77936]\n\t(No symbol) [0x0x7ff669ef9cda]\n\t(No symbol) [0x0x7ff669ed21a1]\n\t(No symbol) [0x0x7ff669f7fc6e]\n\t(No symbol) [0x0x7ff669fa0432]\n\t(No symbol) [0x0x7ff669f786a3]\n\t(No symbol) [0x0x7ff669f41791]\n\t(No symbol) [0x0x7ff669f42523]\n\tGetHandleVerifier [0x0x7ff66a41684d+3059501]\n\tGetHandleVerifier [0x0x7ff66a410c0d+3035885]\n\tGetHandleVerifier [0x0x7ff66a430400+3164896]\n\tGetHandleVerifier [0x0x7ff66a158c3e+185118]\n\tGetHandleVerifier [0x0x7ff66a16054f+216111]\n\tGetHandleVerifier [0x0x7ff66a1472e4+113092]\n\tGetHandleVerifier [0x0x7ff66a147499+113529]\n\tGetHandleVerifier [0x0x7ff66a12e298+10616]\n\tBaseThreadInitThunk [0x0x7ff976d1e8d7+23]\n\tRtlUserThreadStart [0x0x7ff977f3c34c+44]\n\n","statusTrace":"  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features\\steps\\login_steps.py\", line 47, in step_invalid_login\n    context.login_page.enter_username(\"invalid@example.com\")\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\OneDrive\\Desktop\\task18\\features\\pages\\login_page.py\", line 25, in enter_username\n    self.enter_text(self.USERNAME_INPUT, username)\n    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\OneDrive\\Desktop\\task18\\features\\pages\\base_page.py\", line 42, in enter_text\n    element = self.wait_for_element(locator)\n  File \"C:\\Users\\surek\\OneDrive\\Desktop\\task18\\features\\pages\\base_page.py\", line 22, in wait_for_element\n    return self.wait.until(EC.presence_of_element_located(locator))\n           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 137, in until\n    value = method(self._driver)\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\expected_conditions.py\", line 110, in _predicate\n    return driver.find_element(*locator)\n           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 898, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[{"name":"Given I open the login page","time":{"start":1753527203175,"stop":1753527203404,"duration":229},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":0},{"name":"When I enter invalid credentials","time":{"start":1753527203405,"stop":1753527203426,"duration":21},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=138.0.7204.169)\nStacktrace:\n\tGetHandleVerifier [0x0x7ff66a13e935+77845]\n\tGetHandleVerifier [0x0x7ff66a13e990+77936]\n\t(No symbol) [0x0x7ff669ef9cda]\n\t(No symbol) [0x0x7ff669ed21a1]\n\t(No symbol) [0x0x7ff669f7fc6e]\n\t(No symbol) [0x0x7ff669fa0432]\n\t(No symbol) [0x0x7ff669f786a3]\n\t(No symbol) [0x0x7ff669f41791]\n\t(No symbol) [0x0x7ff669f42523]\n\tGetHandleVerifier [0x0x7ff66a41684d+3059501]\n\tGetHandleVerifier [0x0x7ff66a410c0d+3035885]\n\tGetHandleVerifier [0x0x7ff66a430400+3164896]\n\tGetHandleVerifier [0x0x7ff66a158c3e+185118]\n\tGetHandleVerifier [0x0x7ff66a16054f+216111]\n\tGetHandleVerifier [0x0x7ff66a1472e4+113092]\n\tGetHandleVerifier [0x0x7ff66a147499+113529]\n\tGetHandleVerifier [0x0x7ff66a12e298+10616]\n\tBaseThreadInitThunk [0x0x7ff976d1e8d7+23]\n\tRtlUserThreadStart [0x0x7ff977f3c34c+44]\n\n","statusTrace":"  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features\\steps\\login_steps.py\", line 47, in step_invalid_login\n    context.login_page.enter_username(\"invalid@example.com\")\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\OneDrive\\Desktop\\task18\\features\\pages\\login_page.py\", line 25, in enter_username\n    self.enter_text(self.USERNAME_INPUT, username)\n    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\OneDrive\\Desktop\\task18\\features\\pages\\base_page.py\", line 42, in enter_text\n    element = self.wait_for_element(locator)\n  File \"C:\\Users\\surek\\OneDrive\\Desktop\\task18\\features\\pages\\base_page.py\", line 22, in wait_for_element\n    return self.wait.until(EC.presence_of_element_located(locator))\n           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 137, in until\n    value = method(self._driver)\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\support\\expected_conditions.py\", line 110, in _predicate\n    return driver.find_element(*locator)\n           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 898, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\surek\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"hasContent":true,"attachmentsCount":0,"attachmentStep":false,"shouldDisplayMessage":true,"stepsCount":0},{"name":"And I click the login button","time":{"start":1753527205569,"stop":1753527205569,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":0},{"name":"Then I should see an error message","time":{"start":1753527205569,"stop":1753527205569,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"hasContent":false,"attachmentsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":0}],"attachments":[],"parameters":[],"hasContent":true,"attachmentsCount":0,"attachmentStep":false,"shouldDisplayMessage":false,"stepsCount":4},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"Login functionality for GuviZenClass portal"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"8b0b505c94856499.json","parameterValues":[]}